### PR TITLE
86 GITHUB_EMBED

### PR DESCRIPTION
**Pull request by rachelwhitton**
## Summary

Removes `gatsby-remark-github` npm package, removes related config from `gatsby-config.js`, removes setup instructions from the `README.md` file, and replaces instances of usage across the docs with either a crosslink or a manual copy of the code snippets instead of using GITHUB_EMBED: 

* [Core Updates](https://pr-9295-documentation.appa.pantheon.site/core-updates#error-updating-conflict-modifydelete-pantheonupstreamyml-deleted-in-head-and-modified-in-upstreammaster-version-upstreammaster-of-pantheonupstreamyml-left-in-tree)
* [Access Backups](https://pr-9295-documentation.appa.pantheon.site/guides/backups/access-backups)
* [Local Development Configuration](https://pr-9295-documentation.appa.pantheon.site/guides/local-development/configuration#wordpress-wp-config-localphp)
* [Automate Log Downloads](https://pr-9295-documentation.appa.pantheon.site/guides/logs-pantheon/automate-log-downloads#create-a-script)
* [Configure Your wp-config.php File](https://pr-9295-documentation.appa.pantheon.site/guides/php/wp-config-php#pantheons-wordpress-config)
* [Debug Local DNS Cache](https://pr-9295-documentation.appa.pantheon.site/local-dns-cache)
* [Upload Your Files](https://pr-9295-documentation.appa.pantheon.site/guides/manual-d8-composer-to-d8/files)
* [Terminus: Get Started](https://pr-9295-documentation.appa.pantheon.site/terminus/examples#reset-dev-environment-to-live)

### CODE REVIEW

File: source/content/guides/backups/02-access-backups.md [hunk 1]
```diff
@@ -23,7 +23,72 @@ Pantheon backups are stored offsite on Google Cloud Storage instances. We recomm
 
 <Download file="pantheon-backup-to-s3.sh" />
 
-GITHUB-EMBED https://github.com/pantheon-systems/documentation/blob/main/source/scripts/pantheon-backup-to-s3.sh.txt bash GITHUB-EMBED
```
Comment (stevector) [inline source/content/guides/backups/02-access-backups.md:26]: @rachelwhitton should we delete these `.txt` files too?

File: source/content/guides/local-development/02-configuration.md [hunk 2]
```diff
@@ -214,13 +214,7 @@ This file is ignored by the `.gitignore` file in [WordPress](https://github.com/
 
 Pantheon sites that install WordPress 5.5 include a `wp-config-local-sample.php` file. Older sites can copy the [wp-config-local-sample.php](https://github.com/pantheon-systems/WordPress/blob/default/wp-config-local-sample.php) file on GitHub to the same directory as the site's `wp-config.php`, or create one in that location as shown here.
 
-The following can be used as a starting point for `wp-config-local.php`. Replace the database values with the values from your local environment, and the key/salt values with your unique phrase (generated from [WordPress.org](https://api.wordpress.org/secret-key/1.1/salt/)).
-
-<Accordion title={"Full text of wp-config-local-sample.php"} id={"full-wp-config-local-sample"}>
-
-GITHUB-EMBED https://github.com/pantheon-systems/WordPress/blob/default/wp-config-local-sample.php php GITHUB-EMBED
```
Comment (stevector) [inline source/content/guides/local-development/02-configuration.md:221]: @rachelwhitton we should probably retain the link to this file, right?

File: source/content/guides/logs-pantheon/04-automate-log-downloads.md [hunk 3]
```diff
@@ -47,19 +47,153 @@ Read the comments in the scripts carefully to ensure that you modify variables c
 
   <Download file="collect-logs-rsync.sh" />
 
-  GITHUB-EMBED https://github.com/pantheon-systems/documentation/blob/main/source/scripts/collect-logs-rsync.sh.txt shell:title=collect-logs-rsync.sh GITHUB-EMBED
```
Comment (stevector) [inline source/content/guides/logs-pantheon/04-automate-log-downloads.md:50]: Another .txt file to possibly delete

File: source/content/guides/backups/02-access-backups.md (hunk 1 above)
Comment (rachelwhitton) [reply source/content/guides/backups/02-access-backups.md:26]: No because the [download component](https://github.com/pantheon-systems/documentation/blob/main/src/components/download/index.js#L8) expects it to be there for the button to work above the code snippet (used in cases where we are giving a full bash script for the user to run)
Screenshot 2024-11-07 at 1 37 59 PM

File: source/content/guides/local-development/02-configuration.md (hunk 2 above)
Comment (rachelwhitton) [reply source/content/guides/local-development/02-configuration.md:221]: It's linked above on line 215

File: source/content/guides/logs-pantheon/04-automate-log-downloads.md [hunk 4]
```diff
@@ -71,7 +205,7 @@ You can run the script with parameters for reusability.
 
 1. Edit the 'collect-logs-rsync.sh' (you can choose to disable the AGGREGATE_NGINX):
   
-  ```bash{promptUser:user}
```
Comment (stevector) [inline source/content/guides/logs-pantheon/04-automate-log-downloads.md:74]: Is this an unrelated fix? I'm fine with including it if it's generally beneficial.

File: source/content/guides/local-development/02-configuration.md (hunk 2 above)
Comment (rachelwhitton) [reply source/content/guides/local-development/02-configuration.md:221]: actually i made a mistake on this one i didn't mean to delete line 217 i meant to edit it, will fix now

File: source/content/guides/logs-pantheon/04-automate-log-downloads.md (hunk 4 above)
Comment (rachelwhitton) [reply source/content/guides/logs-pantheon/04-automate-log-downloads.md:74]: unrelated, just noticed it was used incorrectly as i was fixing other things on the page

**Review by stevector — CHANGES_REQUESTED** [review]
@rachelwhitton I should review the build when it completes too but I've already added some questions/comments.

**Review by stevector — APPROVED** [review]
